src/pages/Accounts/AddAccountModal.tsx

```diff
@@ -103,7 +104,7 @@ export const AddAccountModal = () => {
 
   const asset = useMemo(() => {
     if (!selectedChainId) return
-    return assets[getChainAdapterManager().get(selectedChainId)!.getFeeAssetId()]
+    return assets?.[getChainAdapterManager().get(selectedChainId)!.getFeeAssetId()]
```

Comment:
this highlights a much deeper problem with our types on asset and market data... not for this PR